www/js/settingsCtrl.js

```diff
@@ -1,3 +1,21 @@
-angular.module('starter').controller("Settings", function ($scope, $stateParams, $state, resourceService) {
+angular.module('starter').controller("Settings", function ($scope, $stateParams, $ionicModal, $state, userInfoService) {
+
+	$ionicModal.fromTemplateUrl('templates/resources.html', {
+        scope: $scope,
+        animation: 'slide-in-up'
+    }).then(function(modal) {
+        $scope.modal = modal;
+    })
+
+	$scope.manage = function() {
+    	$scope.modal.show();
+  	}
+
+  	$scope.$on('$ionicView.enter', function() {
+     // Code you want executed every time view is opened
+     console.log('Opened!')
```

Comment:
Do not commit console logs please! If you are commiting an unfinished feature with an unfinished function, please use `console.error('Not impleneted', ...)`